tools/validate_reflection.py

```diff
@@ -673,7 +677,8 @@ def validate_file(file):
                 fwd_swapped.remove(field)
                 f_index += 1
             else:
-                assert reflect_field == field, "Reflection for %s should have field %s instead of %s or else it should indicate if the field should be ignored (%s) or swapped (%s)" %(reflection_name, field, reflect_field, ignore_str, swap_str)
+                if reflect_field != field:
```

Comment:
Is there a reason to add this if statement?

Reply:
Left over debug code.  I will remove.